.gitignore

```diff
@@ -114,3 +114,8 @@ data/catch-ignore.yml
 release_config.json
 web/userdata.js
 location.json
+
+#Multiple config
+configs/
+!configs/config.json.example
+!configs/release_config.json.example
```

Comment:
but you dont comit in the config and example files, you just delete them from directory with this PR